cmd/skaffold/app/cmd/init.go

```diff
@@ -52,6 +53,7 @@ func NewCmdInit() *cobra.Command {
 			f.BoolVar(&force, "force", false, "Force the generation of the Skaffold config")
 			f.StringVar(&composeFile, "compose-file", "", "Initialize from a docker-compose file")
 			f.StringArrayVarP(&cliArtifacts, "artifact", "a", nil, "'='-delimited Dockerfile/image pair, or JSON string, to generate build artifact\n(example: --artifact='{\"builder\":\"Docker\",\"payload\":{\"path\":\"/web/Dockerfile.web\"},\"image\":\"gcr.io/web-project/image\"}')")
+			f.StringArrayVarP(&cliKubernetesManifests, "kubernetes-manifest", "k", nil, "list of predefined kubectl manifests (overrides detection)")
```

Comment:
Can we add more explanation on if it can accept a file glob, or a directory ?
Do they need to be valid manifests?
```
list of predefined valid kubectl manifests (overrides detection) e.g. deploy/*.yaml, k8s/ or k8s/pod.yaml"
```

Reply:
I'll add more explanation. No, they don't have to be valid kubectl manifests, they can be just a placeholder that is created later.